### DISCUSSION TITLE
can I use `envelop` to replace `apollo-server-micro` & `apollo-server-core`?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [dotansimha/envelop#1430] [Q&A] [answered]
apollo server gives a lot of errors randomly. so can `envelop` be used instead of `apollo`?

anything `apollo` supports that `envelop` doesn't? i probably don't understand this project well enough so curious 🤔

**Comment by n1ru4l** [comment, marked as answer]
Hey @deadcoder0904, envelop is a plugin system for graphql-js that [can be used with a variety of GraphQL servers](https://www.envelop.dev/docs/integrations). If you want to replace apollo-server with another GraphQL server you should look at [GraphQL Yoga](https://www.graphql-yoga.com/), which is our recommended way of using envelop! Yoga is JavaScript platform-agnostic and runs on serverless (e.g. AWS Lambda), the edge (e.g. cloudflare workers), deno, Node.js, and even the browser! [We have a full tutorial for getting started with Yoga](https://www.graphql-yoga.com/tutorial/basic/00-introduction) and [a guide for migrating from Apollo Server to GraphQL Yoga](https://www.graphql-yoga.com/docs/migration/migration-from-apollo-server).

**Reply by deadcoder0904** [reply]
yeah was just looking at all the guild products. thought yoga was abandoned by prisma but looks like its back on track. thank you for all the awesome work!
